hexrdgui/masking/mask_manager_dialog.py

```diff
@@ -427,3 +435,49 @@ def apply_changes(self):
             self.change_mask_presentation(index, name)
         self.changed_masks = {}
         self.ui.apply_changes.setEnabled(False)
+
+    def selected_changed(self):
+        selected = self.ui.masks_tree.selectedItems()
+        self.ui.presentation_selector.setEnabled(len(selected) > 1)
+        self.ui.export_selected.setEnabled(len(selected) > 1)
+        self.ui.remove_selected.setEnabled(len(selected) > 1)
```

Comment:
I would typically do something like this:

```python
        enable = len(selected) > 1
        widgets = [
            self.ui.presentation_selector,
            self.ui.export_selected,
            self.ui.remove_selected,
        ]
        for w in widgets:
            w.setEnabled(enable)
```

But you do not have to change it!